mftf/2.3/versioning.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Versioning Policy for MFTF
+---
+
+This documemt describes the versioning policy for the Magento Functional Testing Framework (MFTF), including the version numbering schema.
+
+## Backwards Compatibility Definition
+
+Backwards compatibility is when a test undergoes changes, but achieves the same results as before and remains compatible with potential test customizations.
+
+Different types of changes that are backwards compatible:
+
+- **Test Flow change (Test/ActionGroup)** - A backwards compatible modification of a test flow would not diminish the original set of actions in the test.
+Some modifications may change an action's sequence or behavior, but still allows an extension to achieve the same test results without changing the test extension.
+- **Test Entity change (Data/Section/Page/Metadata)** - Adding new values or updating a `value` of existing entities which do **NOT** require updates to the test.
+- **Test Annotation change** - Annotations can be changed and are considered backward compatible changes.
+However, removing or changing a `<group />` annotation is considered a backward incompatible change.
+- Changes which delete and/or rename (Test/Action Group/Data/Metadata/Page/Section/Action)'s id attribute are considered as a backward incompatible change. Changing a reference to a data entity is considered a backward incompatible change.
+
```

Comment:
Suggestion -- tie the backward compatibility section to the versioning schema instead of calling it out as a definition. That is,  the versioning schema is tied to the type of changes included in test updates to be packaged for release....

Also -- backwards ==> backward

For example, the versioning schema is tied to the types of changes included in a test update.  Then, mention two types of changes -- backward compatible with examples, and backward incompatible.

Reply:
This section discussed changes to tests rather than MFTF. Now that I know that distinction, this section belonged elsewhere and I removed it.